test/external.jl

```diff
@@ -45,4 +48,7 @@ close(source_file)
 # @test read(group2["1"])==1
 # @test read(group2["1.1"])==1.1
 
+rm(fn1)
+# rm(fn2)
```

Comment:
@ggggggggg trying to rm(fn2) here results is a `unlink: resource busy or locked (EBUSY)` error even though the handle has been explicitly closed, any ideas here?